### ISSUE TITLE
define kmer during assembly

### ISSUE THREAD

**Issue by ganiatgithub** [bxlab/metaWRAP#242]
Hi,

I was wondering if it is possible to use my own sets of kmers during assembly?

Cheers,
Gaofeng

**Comment by ursky** [comment]
No, but if you are diving this deeply into assembly parameterization then you should probably be running the metaspades/megahit assemblies manually as opposed to using the wrappper :)
